R/calcOutput.R

```diff
@@ -89,6 +89,7 @@ calcOutput <- function(type,aggregate=TRUE,file=NULL,years=NULL,round=NULL,suppl
     # rename column names from old to new convention, if necessary
     if(any(names(rel[[r]])=="CountryCode")) names(rel[[r]])[names(rel[[r]])=="CountryCode"] <- "country"
     if(any(names(rel[[r]])=="RegionCode")) names(rel[[r]])[names(rel[[r]])=="RegionCode"] <- "region"
+    if(any(names(rel[[r]])=="ExtraRegions")) names(rel[[r]])[names(rel[[r]])=="ExtraRegions"] <- "extra"
```

Comment:
The lines with countryCode and RegionCode are only needed for backwards compatibility. As the ExtraRegions approach is new I would like you ask you to remove this line and useinstead the new terminology from the beginning

Reply:
A comment in the code that this is only for backwards compatibility could have helped.